File: src/main/java/no/nav/syfo/domain/Sykepengesoknad.java
package no.nav.syfo.domain;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;

import java.time.LocalDate;

@Value
@Builder
@Getter
public class Sykepengesoknad {
    private String uuid;
    private String status;
    private String saksId;
    private String oppgaveId;
    private String journalpostId;
    private LocalDate fom;
    private LocalDate tom;
}

File: src/main/kotlin/no/nav/syfo/domain/Sykepengesoknad.kt
package no.nav.syfo.domain

import java.time.LocalDate

data class Sykepengesoknad(
    val uuid: String? = null,
    val status: String? = null,
    val saksId: String? = null,
    val oppgaveId: String? = null,
    val journalpostId: String? = null,
    val fom: LocalDate? = null,
    val tom: LocalDate? = null
)
